conf.json

```diff
@@ -99,7 +99,9 @@
         "acl is_turbovote_staging hdr_end(host) -i turbovote.net",
         "use_backend turbovote_staging if is_turbovote_staging",
         "acl is_online_voter_reg_api hdr_end(host) -i online-voter-reg-api.turbovote.org",
-        "use_backend online_voter_reg_api if is_online_voter_reg_api"
+        "use_backend online_voter_reg_api if is_online_voter_reg_api",
+        "acl is_user_http_api base -i ^api[.]turbobote[.]org/users.*",
```

Comment:
Would this be faster?

`acl is_user_http_api base_beg -i api.turbovote.org/users`